crates/turborepo-lib/src/cli/mod.rs

```diff
@@ -758,6 +761,192 @@ impl RunArgs {
+                    DryRunMode::Json => "json",
+                },
+                EventType::NonSensitive,
+            );
+        }
+
+        if self.cache_workers != DEFAULT_NUM_WORKERS {
+            telemetry.track_arg_value("cache-workers", self.cache_workers, EventType::NonSensitive);
+        }
+
+        if let Some(concurrency) = &self.concurrency {
+            telemetry.track_arg_value("concurrency", concurrency, EventType::NonSensitive);
+        }
+
+        if !self.global_deps.is_empty() {
+            telemetry.track_arg_value("global-deps", self.cache_workers, EventType::NonSensitive);
+        }
+
+        if let Some(graph) = &self.graph {
+            telemetry.track_arg_value("graph", graph, EventType::NonSensitive);
+        }
+
+        if self.env_mode != EnvMode::default() {
+            telemetry.track_arg_value(
+                "env-mode",
+                match self.env_mode {
+                    EnvMode::Infer => "infer",
+                    EnvMode::Loose => "loose",
+                    EnvMode::Strict => "strict",
+                },
+                EventType::NonSensitive,
+            );
+        }
+
+        if let Some(output_logs) = &self.output_logs {
+            telemetry.track_arg_value(
+                "output-logs",
+                match output_logs {
+                    OutputLogsMode::Full => "full",
+                    OutputLogsMode::None => "none",
+                    OutputLogsMode::HashOnly => "hash-only",
+                    OutputLogsMode::NewOnly => "new-only",
+                    OutputLogsMode::ErrorsOnly => "errors-only",
+                },
+                EventType::NonSensitive,
+            );
+        }
+
+        if self.log_order != LogOrder::default() {
+            telemetry.track_arg_value(
+                "log-order",
+                match self.log_order {
+                    LogOrder::Auto => "auto",
+                    LogOrder::Stream => "stream",
+                    LogOrder::Grouped => "grouped",
+                },
+                EventType::NonSensitive,
+            );
+        }
+
+        if self.log_prefix != LogPrefix::default() {
+            telemetry.track_arg_value(
+                "log-prefix",
+                match self.log_prefix {
+                    LogPrefix::Auto => "auto",
+                    LogPrefix::None => "none",
+                    LogPrefix::Task => "task",
+                },
+                EventType::NonSensitive,
+            );
+        }
+
+        // track sizes
+        if !self.filter.is_empty() {
+            telemetry.track_arg_value("filter", self.filter.len(), EventType::NonSensitive);
+        }
+
+        if !self.scope.is_empty() {
+            telemetry.track_arg_value("scope", self.scope.len(), EventType::NonSensitive);
+        }
+
+        if !self.ignore.is_empty() {
+            telemetry.track_arg_value("ignore", self.ignore.len(), EventType::NonSensitive);
+        }
```

Comment:
These vals would be sensitive and hashed vals are useless here so I've opted to track the size of the respective item here to give an idea of the scope of usage. 

For some of these (filter) we can add more events deeper into the code to get a better idea about the type of items being filtered on.

Reply:
Do we want to change the arg name to something like `filter-len` or `num-of-filters` to make it clear in the data that the user isn't passing `--filter=3`?

Reply:
Good call, done!